modules/advangelistsBidAdapter.js

```diff
@@ -0,0 +1,443 @@
+  try {
+    return window.top.document.referrer;
+  } catch (e) {
+    return '';
+  }
+}
+
+function getVideoTargetingParams(bid) {
+  return Object.keys(Object(bid.params.video))
+    .filter(param => includes(VIDEO_TARGETING, param))
+    .reduce((obj, param) => {
+      obj[ param ] = bid.params.video[ param ];
+      return obj;
+    }, {});
+}
+
+function createVideoRequestData(bid, bidderRequest) {
+  let topLocation = getTopWindowLocation(bidderRequest);
+  let topReferrer = getTopWindowReferrer();
+
+  let sizes = getVideoSizes(bid);
+  let firstSize = getFirstSize(sizes);
+
+  let video = getVideoTargetingParams(bid);
+  var o = {
+    'device': {
+      'langauge': (global.navigator.language).split('-')[0],
+      'dnt': (global.navigator.doNotTrack === 1 ? 1 : 0),
+      'devicetype': isMobile() ? 4 : isConnectedTV() ? 3 : 2,
+      'js': 1,
+      'os': getOsVersion()
+    },
+    'at': 2,
+    'site': {},
+    'tmax': 3000,
+    'cur': ['USD'],
+    'id': bid.bidId,
+    'imp': [],
+    'regs': {
+      'ext': {
+      }
+    },
+    'user': {
+      'ext': {
+      }
+    }
+  };
+
+  o.site['page'] = topLocation.href;
+  o.site['domain'] = topLocation.hostname;
+  o.site['search'] = topLocation.search;
+  o.site['domain'] = topLocation.hostname;
+  o.site['ref'] = topReferrer;
+  o.site['mobile'] = isMobile() ? 1 : 0;
+  var secure = topLocation.protocol.indexOf('https') === 0 ? 1 : 0;
+
+  o.device['dnt'] = getDoNotTrack() ? 1 : 0;
+
+  findAndFillParam(o.site, 'name', function() {
+    return global.top.document.title;
+  });
+
+  findAndFillParam(o.device, 'h', function() {
+    return global.screen.height;
+  });
+  findAndFillParam(o.device, 'w', function() {
+    return global.screen.width;
+  });
+
+  let placement = getVideoBidParam(bid, 'placement');
+
+  for (let j = 0; j < sizes.length; j++) {
+    o.imp.push({
+      'id': '' + j,
+      'displaymanager': '' + BIDDER_CODE,
+      'displaymanagerver': '' + ADAPTER_VERSION,
+      'tagId': placement,
+      'bidfloor': 2.0,
+      'bidfloorcur': 'USD',
+      'secure': secure,
+      'video': Object.assign({
+        'id': utils.generateUUID(),
+        'pos': 0,
+        'w': firstSize.w,
+        'h': firstSize.h,
+        'mimes': DEFAULT_MIMES
+      }, video)
+
+    });
+  }
+
+  if (bidderRequest && bidderRequest.gdprConsent) {
+    let { gdprApplies, consentString } = bidderRequest.gdprConsent;
+    o.regs.ext = {'gdpr': gdprApplies ? 1 : 0};
+    o.user.ext = {'consent': consentString};
+  }
+
+  return o;
+}
+
+function getTopWindowLocation(bidderRequest) {
+  let url = bidderRequest && bidderRequest.refererInfo && bidderRequest.refererInfo.referer;
+  return parseUrl(config.getConfig('pageUrl') || url, { decodeSearchAsString: true });
+}
+
+function createBannerRequestData(bid, bidderRequest) {
+  let topLocation = getTopWindowLocation(bidderRequest);
+  let topReferrer = getTopWindowReferrer();
+
+  let sizes = getBannerSizes(bid);
+
+  var o = {
```

Comment:
change `var` to `const` since it is not being reassigned

Reply:
Thanks. Replace var with const